sky/serve/replica_managers.py

```diff
@@ -539,6 +571,14 @@ def scale_up(self,
         self._next_replica_id += 1
 
     def _terminate_replica(self, replica_id: int, sync_down_logs: bool) -> None:
+        p = self._launch_process_pool[replica_id]
```

Comment:
```suggestion
        launch_process = self._launch_process_pool[replica_id]
```
to avoid re-assign this variable